### PR TITLE
Update electric, far, logisim, pmd and sejda_console packages

### PR DESCRIPTION
**Pull request by uberhacker**
Updates to change dependencies from jdk8 to openjdk8.

### CODE REVIEW

File: packages/pmd.rb [hunk 1]
```diff
@@ -3,45 +3,45 @@
...
 
+  no_compile_needed
+
+  def self.build
+    cpd = <<~EOF
+      #!/bin/bash
+      PWD=$(pwd)
+      cd #{CREW_PREFIX}/share/pmd
+      bin/run.sh cpd "$@"
+      cd $PWD
```
Comment (supechicken) [inline packages/pmd.rb:23]: We don't need to `cd` back since this is independent from the shell process

File: packages/pmd.rb [hunk 2]
```diff
@@ -3,45 +3,45 @@
...
   depends_on 'unzip'
 
+  no_compile_needed
+
+  def self.build
+    cpd = <<~EOF
+      #!/bin/bash
+      PWD=$(pwd)
+      cd #{CREW_PREFIX}/share/pmd
+      bin/run.sh cpd "$@"
```
Comment (supechicken) [inline packages/pmd.rb:22]: `exec bin/run.sh cpd "$@"` to prevent bash waiting in background?

File: packages/pmd.rb (hunk 1 above)
Comment (uberhacker) [reply packages/pmd.rb]: Good call.  Fixed.

File: packages/pmd.rb (hunk 2 above)
Comment (uberhacker) [reply packages/pmd.rb]: Fixed.

**Review by supechicken — APPROVED** [review]

**Review by satmandu — APPROVED** [review]

**Pull request merged**